rules_ruby_gapic/ruby_gapic.bzl

```diff
@@ -56,17 +56,40 @@ def ruby_gapic_library(
 #
 # name: name of the rule
 # srcs: proto files wrapped in the proto_library rule
-# ruby_cloud_params a string-string dictionary of the cloud generator parameters
-#   (e.g. :gem.:name)
-# grpc_service_configs: a list of labels of the grpc service configs (or an empty list)
+# ruby_cloud_title: cloud gem's title. 
+#   It is separated from the extra_protoc_parameters because of issues with build_gen: https://github.com/googleapis/gapic-generator-ruby/issues/539
+# ruby_cloud_description: cloud gem's description. 
+#   It is separated from the extra_protoc_parameters because of issues with build_gen: https://github.com/googleapis/gapic-generator-ruby/issues/539
+# extra_protoc_parameters: a string-string dictionary of the cloud generator parameters
+#   (e.g. ruby-cloud-gem-name)
+# grpc_service_config: a label to the grpc service config
 #
 def ruby_cloud_gapic_library(
   name,
   srcs,
+  ruby_cloud_title = "",
+  ruby_cloud_description = "",
   extra_protoc_parameters = [],
   grpc_service_config = None,
   **kwargs):
   
+  if extra_protoc_parameters:
+    for key_val_string in extra_protoc_parameters:
+        key_val_split = key_val_string.split("=", 1)
```

Comment:
do you want to trim the key/value pairs as well (to accommodate for leading and trailing whitespaces)?

Reply:
that's a good idea